src/stores/navigated.ts

```diff
@@ -0,0 +1,31 @@
+import reactMinistore from './react-ministore'
+import scrollTopStore from './scrollTop'
+
+// Keep track of whether a programmatic scroll will occur in order to ignore events.
+let isAutoScrolling = false
+
+/**
+ * Prepares the navigatedStore to ignore scroll events that are triggered programmatically.
+ */
+export const preventScrollDetection = () => {
+  isAutoScrolling = true
+  setTimeout(() => {
+    isAutoScrolling = false
+  }, 200)
```

Comment:
This timeout should probably be persisted and cancelled at the beginning of the function. Otherwise two calls within 200ms will cause `isAutoScrolling` to flip back-and-forth.

The timeout should probably also be cancelled when the user navigates.